corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -105,6 +113,7 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None):
             if getattr(f, 'schedule', False) and f.schedule.enabled
         ])
 
+    include_fullstory = _form_uses_case(module, form) and not _form_too_large(app, form)
```

Comment:
This can go in the if statement above; I don't think we care about fullstory for apps that aren't using vellum case management.